lib/scrabble.rb

```diff
@@ -15,4 +25,5 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+  total
```

Comment:
This total here is outside of a method, but inside the class. It is disconnected from the local variable `total` on lines 11 and 13.